tests/api/test_patterns.py

```diff
@@ -231,3 +234,73 @@ def test_bimonthly_every_two_months_two_levels(
     # test preview
     issues = holding.prediction_issues_preview(13)
     assert issues[-1] == 'Jg 57 Nr 3 Mai 2026'
+
+
+def test_pattern_preview_api(
```

Comment:
Is it possible to add error tests?

Reply:
it is already tested in the function `test_patterns_functions`